front/src/store/modules/tickets.js

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+const { serverPath } = require('@/config/config.dev.json');
+
+export default {
+  namespaced: true,
+  state: {
+    tickets: []
+  },
+  mutations: {
+    SET_USER_TICKETS (state, tickets) {
+      state.tickets = tickets;
+    }
+  },
+  actions: {
+    refreshTickets ({ commit }) {
+      axios
+        .get(`${serverPath}/api/tickets`)
+        .then(({ data }) => {
+          commit('SET_USER_TICKETS', data);
+        });
+    }
+  },
+  getters: {
+    tickets (state) {
+      return state.tickets;
+    }
+  }
+};
```

Comment:
please, provide correct file name for store module
https://github.com/horba/my-trip/wiki/Frontend-Namings-and-Structure